src/genophenocorr/analysis/_config.py

```diff
@@ -140,15 +124,21 @@ def pval_correction(self, pval_correction: typing.Optional[str]):
         if pval_correction in P_VAL_OPTIONS:
             self._pval_correction = pval_correction
         else:
-            self._logger.warning('Ignoring invalid `pval_correction` value %s. Not doing p-value correction.', pval_correction)
+            self._logger.warning('Ignoring invalid `pval_correction` value %s. Using default "bonferroni" correction.', pval_correction)
```

Comment:
@lnrekerle I hate myself for this but I cannot stay quiet. This is still not right. Consider this situation:

```python
config = CohortAnalysisConfiguration.builder()
  .pval_correction('sidak')
  .pval_correction('EVIL HACK')
  .build()
```
The code above will log `EVIL_HACK` and advertise falling back to `bonferroni`, while `sidak` is actually what will be done!
So, you must log the actual value instead of `bonferroni`.

---
EDIT: I think this will also apply to other changes in this builder. Please revise accordingly.

Reply:
I think the new messages are better. Let me know what you think.

Reply:
Looks good, thanks!